tests/gordo_components/client/test_client.py

```diff
@@ -443,90 +440,52 @@ def test_ml_server_dataframe_to_dict_and_back(tags: typing.List[str]):
         assert np.allclose(df[top_lvl_name].values, df_clone[top_lvl_name].values)
 
 
+def _machine(name: str) -> Machine:
+    """
+    Helper to build a basic Machine, only defining its name
+    """
+    # TODO: Fix this from changes from EndpointMetadata being removed
```

Comment:
Are these fixes still needing to be done?

Reply:
Same as previous comment, soon will not be relevant.